test/distributed/fsdp/test_fsdp_state_dict.py

```diff
@@ -715,7 +715,7 @@ def test_state_dict_with_ignored_modules(self, prefix, ignore_inner):
                 param.zero_()
 
         to_load = {k[len(prefix_str):] : v for k, v in sd1.items()}
-        nonwrapped_model.load_state_dict(to_load)
+        nonwrapped_model.load_state_dict(to_load, strict=True)
```

Comment:
Is it intentional to add `strict=True` for the non-wrapped model too?

Reply:
It wasn't, but this doesn't make a difference since `strict=True` is the default anyways.